src/docs/product/metrics/index.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: "Metrics"
+sidebar_order: 55
+description: "Learn about Sentry's Developer Metrics, which help you track and visualize the data points you care about over time, in order to monitor your application health and identify issues."
+---
+
+<Include name="feature-stage-alpha-metrics.mdx" />
+
+Sentry metrics help you measure the data points you care about over time, in order to pinpoint and solve issues impacting user experience and app performance. Track everything from processing time and event size to user signups and conversion rates, and correlate them with powerful tracing data in order to get deeper insights and solve issues faster.
+
+![Metrics UI](Index.png)
+
+## What Are Metrics?
+
+Metrics are numerical values that help you track the things you care about most in your application over time. You can create metrics to measure things like latency, processing time or user signups, and visualise them in Sentry in order to get a comprehensive understanding of the health of your systems.
+
+## Metric Types
+
+Metrics at Sentry come in different flavors, in order to help you track your data in the most efficient and cost-effective way. The types of metrics we currently support are:
+
+- Counters → track a value that can only be incremented (e.g. button clicks)
+- Distributions → track a list of values over time in on which you can perform aggregations like max, min, avg (e.g. page load times)
+- Gauges → track a value that can go up and down (e.g. available disk space, memory used)
+- Sets → track a set of values on which you can perform aggregations such as count_unique (e.g. number of unique users)
+
+A metric also needs to have a unit associated to it, so that you know what you're dealing with. Examples of units are seconds, milliseconds, bytes, or even potatoes if you like.
+
+## Augmenting Metrics with Tags
+
+Metrics are pretty powerful, but you can enrich them further by adding dimensions to them in the form of Tags. They are key/value string pairs (e.g. cuisine:Italian) that are associated with metrics to provide contextual information, distinguish, and group metrics during analysis.
```

Comment:
```suggestion
Metrics are powerful on their own, but you can enrich them further by adding dimensions in the form of Tags. These are key/value string pairs (for example, `cuisine:Italian`) that are associated with metrics to provide contextual information, and are often used to filter and group them during analysis.
```

Reply:
@lizokm I'm accepting the suggestion, I've just removed the link to the Tags definition we have in issues since they work slightly differently on metrics today than they do on events. We will be working to unify this in the next weeks and can reference the same page.